src/__tests__/Video.test.tsx

```diff
@@ -0,0 +1,26 @@
+import Video from '../Video/index';
+import { NativeModules } from 'react-native';
+
+jest.mock(
+  '../../node_modules/react-native/Libraries/EventEmitter/NativeEventEmitter'
+);
+
+jest.mock('uuid', () => ({
```

Comment:
I think this might no longer be working with the recent change to how uuids were being generated, but it doesn't look like tests are currently running on CI, so I can confirm and fix up.